File: class_4/class_4/app/src/main/java/com/alejouribesanchez/class_4/Main5Activity.java
package com.alejouribesanchez.class_4;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.RelativeLayout;

public class Main5Activity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main5);

        RelativeLayout relativeLayout = (RelativeLayout)findViewById(R.id.activity_main5);
        relativeLayout.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(Main5Activity.this, Main6Activity.class);
                startActivity(intent);
            }
        });
    }
}

File: class_4/class_4/app/src/main/java/com/alejouribesanchez/class_4/Main5Activity.kt
package com.alejouribesanchez.class_4

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.RelativeLayout

class Main5Activity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main5)

        val relativeLayout = findViewById(R.id.activity_main5) as RelativeLayout
        relativeLayout.setOnClickListener {
            val intent = Intent(this@Main5Activity, Main6Activity::class.java)
            startActivity(intent)
        }
    }
}
